sample/kotlin/src/main/kotlin/com/datadog/android/sample/data/DataRepository.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sample.data
+
+import com.datadog.android.sample.data.model.Log
+import com.datadog.android.sample.data.model.LogsCollection
+import retrofit2.Call
+import retrofit2.Response
+
+class DataRepository(private val remoteDataSource: RemoteDataSource) {
+
+    fun getPosts(query: String, callback: Callback<List<Log>>) {
```

Comment:
The name `getPosts` doesn't match what you're doing (you're fetching logs, not posts, evn though one might argue that you need a log to make a post :D )

Reply:
Oh...left over from the first code :)